modules/nf-core/tiddit/sv/meta.yml

```
@@ -21,14 +21,24 @@ input:
       type: file
       description: BAM/CRAM file
       pattern: "*.{bam,cram}"
-  - index:
+  - input_index:
       type: file
       description: BAM/CRAM index file
       pattern: "*.{bai,crai}"
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
   - fasta:
       type: file
       description: Input FASTA file
       pattern: "*.{fasta,fa}"
+  - meta3:
+      type: map
+      description: |
+        Groovy Map containing sample information from bwa index
+        e.g. [ id:'test', single_end:false ]
```

```suggestion
        e.g. `[ id:'test', single_end:false ]`
```
please double check if this meta map example make sense in their context.